src/main/resources/application.yml

```diff
@@ -35,6 +35,7 @@ sqs:
 
 api:
   print-api:
+    base.url: ${API_PRINT_API_BASE_URL}
```

Comment:
Please update `README` if not done already

Reply:
Darn it! I even thought to myself "I must update the readme otherwise Vishal will burn me for it!" 🤣 
Doing it now 👍